### ISSUE TITLE
Add a method to set attributes on tbody

### ISSUE THREAD

**Issue by Fran-Rg** [plumdog/flask_table#135]
The current tbody element is created without element passed to it:
https://github.com/plumdog/flask_table/blob/master/flask_table/table.py#L134

It can be practical to add a ```tbody_attrs = {"class": "text-xs"}``` class

**Comment by plumdog** [comment]
Yep, I reckon something like `tbody_html_attrs` would fit in with https://github.com/plumdog/flask_table#table-configuration-and-options

Are you able to open a PR?
